src/rust/src/functions/lazy.rs

```diff
@@ -113,6 +113,32 @@ pub fn dtype_cols(dtypes: ListSexp) -> Result<PlRExpr> {
     Ok(dsl::dtype_cols(&dtypes).into())
 }
 
+#[savvy]
+pub fn index_cols(indices: NumericSexp) -> Result<PlRExpr> {
+    let n: Vec<i64> = indices
+        .as_slice_i32()?
+        .iter()
+        .map(|x| *x as i64)
+        .collect::<Vec<_>>();
```

Comment:
Please use this:
https://github.com/eitsupi/neo-r-polars/blob/7dea582e0afa34b7b2fc654f5d8a6477c90504aa/src/rust/src/conversion/mod.rs#L242